android/sdl_android/src/main/java/com/smartdevicelink/managers/video/VideoStreamManager.java

```diff
@@ -274,6 +324,26 @@ public void onError(String info) {
         }
     }
 
+    /**
+     * Starts streaming a remote display to the module if there is a connected session. This method of streaming requires the device to be on API level 19 or higher
+     *
+     * @param context            a context that can be used to create the remote display
+     * @param remoteDisplayClass class object of the remote display. This class will be used to create an instance of the remote display and will be projected to the module
+     * @param parameters         streaming parameters to be used when streaming. If null is sent in, the default/optimized options will be used.
+     *                           If you are unsure about what parameters to be used it is best to just send null and let the system determine what
+     *                           works best for the currently connected module.
+     * @param encrypted         a flag of if the stream should be encrypted. Only set if you have a supplied encryption library that the module can understand.
+     * @param landscapeRange    constraints for vehicle display : aspect ratio, min/max resolutions, max diagonal size.
+     * @param portraitRange     constraints for vehicle display : aspect ratio, min/max resolutions, max diagonal size.
+     */
+    public void startRemoteDisplayStream(Context context, Class<? extends SdlRemoteDisplay> remoteDisplayClass, VideoStreamingParameters parameters, final boolean encrypted, VideoStreamingRange landscapeRange, VideoStreamingRange portraitRange) {
```

Comment:
Hello @kostyaBoss,
I want to make sure that the documentation surrounding this method is clear to the developer. Please see the documentation below, to better align with the iOS library I also recommend change the parameter names from `landscapeRange` and `portraitRange` to `supportedLandscapeStreamingRange` and `supportedPortraitStreamingRange`

```
/**
     * Starts streaming a remote display to the module if there is a connected session. This method of streaming requires the device to be on API level 19 or higher
     * Two ranges (supportedLandscapeStreamingRange and supportedLandscapeStreamingRange) can be provided with image dimension ranges and aspect ratio ranges that your remoteDisplay class supports.
     * If the module's screen size for your app changes during streaming (i.e. to a collapsed view, split screen, preview mode, or picture-in-picture), your remoteDisplay will be resized to the new screen size.
     * If either range is `null`, the default is to support all streaming ranges of that format (landscape or portrait).
     * If you wish to disable support for streaming in a given format (landscape or portrait), set a VideoStreamingRange with all `0` values.
     *
     * NOTE If both supportedLandscapeStreamingRange and supportedLandscapeStreamingRange are disabled then the video will not stream.
     *
     * Any changes to screen size will notify the onViewResized method you have implemented in your remoteDisplay class.
     *
     *
     * @param context            a context that can be used to create the remote display
     * @param remoteDisplayClass class object of the remote display. This class will be used to create an instance of the remote display and will be projected to the module
     * @param parameters         streaming parameters to be used when streaming. If null is sent in, the default/optimized options will be used.
     *                           If you are unsure about what parameters to be used it is best to just send null and let the system determine what
     *                           works best for the currently connected module.
     * @param encrypted         a flag of if the stream should be encrypted. Only set if you have a supplied encryption library that the module can understand.
     * @param supportedLandscapeStreamingRange      constraints for vehicle display : min/max aspect ratio, min/max resolutions, max diagonal size.
     * @param supportedLandscapeStreamingRange      constraints for vehicle display : min/max aspect ratio, min/max resolutions, max diagonal size.
     */
```